examples/adaptive_library/DXFImport.py

```diff
@@ -0,0 +1,205 @@
+"""
+Custom DXF Geometry
+===================
+This script applies the adaptive templates functionality to import custom rotor geometry from a
+.dxf file.
+"""
+# %%
+# .. note::
+#    For more information on how to add imported DXF geometries to Adaptive Templates in Motor-CAD,
+#    see :ref:`ref_adaptive_templates_UG` in the :ref:`ref_user_guide`.
+#
+# .. note::
+#    DXF Import for Adaptive Templates in Motor-CAD requires v2024.2.0.6 (Motor-CAD 2024 R2) or
+#    later and PyMotorCAD v0.4.2. To update PyMotorCAD in Motor-CAD, go to
+#    **Scripting -> Settings** and select 'Update to Latest Release' under PyMotorCAD updates.
+
+# %%
+# This script is designed to be run from Motor-CAD template "e8". The "e8" template file contains an
+# imported custom geometry from a .dxf file. The template should be adjusted slightly to more
+# closely match the imported custom geometry before running the Adaptive Templates script.
+# The following Standard Template geometry changes should be made:
+#
+# * Set L1 Mag Gap Inner and L2 Mag Gap Inner to 0 mm (for Magnet Layers 1 and 2).
```

Comment:
Use bold for all UI elements and double backticks for what is entered. Given that I don't know what this looks like, I'll let you make these changes!